conf/application.conf

```diff
@@ -176,3 +176,11 @@ microservice {
 webchat {
   id = TT55004894
 }
+
+tamc-frontend {
+    host="http://localhost:9900"
+}
+
+accessibility-statement {
+    url="http://localhost:12346"
```

Comment:
Port 12346? Is this a dummy port or should this be 9900?

Reply:
Port 123456 is the local port used for accessibility-statement service, I'll be adding configuration changes for Prod/staging shortly

Reply:
12346, but yes - https://github.com/hmrc/accessibility-statement-frontend/blob/master/build.sbt#L52